### PR TITLE
gladtex: add page

### PR DESCRIPTION
**Pull request by Sadeeed**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

issue: #7560

### CODE REVIEW

**Comment by kbdharun** [comment]
@Sadeeed Thanks for your contribution, can you sign the CLA for the PR to go through?

**Comment by Sadeeed** [comment]
@kbdharun signed, thanks

File: pages/common/gladtex.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`gladtex {{input_file.htex}} -r {{resolution}}`
+
+- Keep latex files after conversion:
+
+`gladtex {{input_file.htex}} -k`
+
+- Set background and foreground color of the images:
+
+`gladtex {{input_file.htex}} -b {{background_color}} -f {{foreground_color}}`
```
Comment (reinhart1010) [inline pages/common/gladtex.md:28]:
```suggestion
`gladtex {{path/to/input.htex}} -b {{background_color}} -f {{foreground_color}}`
```

File: pages/common/gladtex.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Convert markdown to html using pandoc:
+
+`pandoc -s -t html --gladtex {{file.md}} | gladtex -o {{file.html}}`
```
Comment (reinhart1010) [inline pages/common/gladtex.md:32]:
```suggestion
`pandoc -s -t html --gladtex {{path/to/input.md}} | gladtex -o {{path/to/output.html}}`
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks for signing! Here are some note from mine to review:

+ It's okay to use `{{path/to/input.htex}}` and so
+ We prefer to use **command mnemonics** for command lines with short parameters (e.g. `-h` for `[h]elp`)

File: pages/common/gladtex.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# gladtex
+
+> A LaTeX formula preprocessor for HTML files. Converts LaTeX formulas to images.
```
Comment (kbdharun) [inline pages/common/gladtex.md:3]:
```suggestion
> A LaTeX formula preprocessor for HTML files.
> It converts LaTeX formulas to images.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Pull request merged**